src/Entity/Timeline/Manifesto.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace AppBundle\Entity\Timeline;
+
+use A2lix\I18nDoctrineBundle\Doctrine\ORM\Util\Translatable;
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\AbstractTranslatableEntity;
+use AppBundle\Entity\AlgoliaIndexedEntityInterface;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="timeline_manifestos")
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\Timeline\ManifestoRepository")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Manifesto extends AbstractTranslatableEntity implements AlgoliaIndexedEntityInterface
+{
+    use Translatable;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="bigint")
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     *
+     * @Algolia\Attribute
+     */
+    private $id;
+
+    public function __construct()
+    {
+        $this->translations = new ArrayCollection();
+    }
+
+    public function __toString()
+    {
+        /** @var ProfileTranslation $translation */
+        if ($translation = $this->translate()) {
+            return $translation->getTitle();
+        }
+
+        return '';
+    }
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @Algolia\Attribute
+     */
+    public function titles(): array
+    {
+        /** @var ProfileTranslation $french */
+        if (!$french = $this->translate('fr')) {
```

Comment:
maybe you can put this code in `__construct()` method and create properties `$french` and `$english` in order to no repeat it in `slugs()` method

Reply:
made a generic method in the abstract class :+1: